src/media-event-filter.ts

```diff
@@ -114,7 +116,7 @@ export const getMediaEventFilter = ({
     ...initialState,
   };
 
-  const isNotReady = (): boolean => state.loading;
+  const isNotReady = (): boolean => allowResumeAfterEnded ? state.loading : state.loading || state.ended;
```

Comment:
Isn't this the wrong way around?

Reply:
If `allowResumeAfterEnded` is `true` we want to allow triggering of events i.e only checking `state.loading`. If `false` we want to have the same behaviour as before.

Reply:
Or maybe I'm wrong

Reply:
if `allowResumeAfterEnded` ( or maybe `allowPlaybackAfterEnded` is a clearer name 🤔 ) is `true` we want the new behavior, if false ( default ) we want the old behavior.

Reply:
I'm a dumb dumb, misunderstood the code 🤦